src/components/HeaderBlock/HeaderBlock.js

```diff
@@ -246,12 +259,31 @@ class HeaderBlock extends React.Component<Props> {
     );
   };
 
+  renderUser = () => {
```

Comment:
I'm guessing conflict wasn't solved in a right way ;/

I've removed this in my previous [PR](https://github.com/pillarwallet/pillarwallet/pull/1714) and added it to Home screen.
Due to the fact, that this component is no longer visible in multiple screens, so it is better to pass it in as a custom side element and remove connection to global state.